Source/LDSwiftEventSource.swift

```diff
@@ -159,7 +182,7 @@ class EventSourceDelegate: NSObject, URLSessionDataDelegate {
                                     timeoutInterval: self.config.idleTimeout)
         urlRequest.httpMethod = self.config.method
         urlRequest.httpBody = self.config.body
-        urlRequest.setValue(self.lastEventId, forHTTPHeaderField: "Last-Event-ID")
+        urlRequest.setValue(self.lastEventId, forHTTPHeaderField: "Last-Event-Id")
```

Comment:
Unrelated fix after regenerating test manifest for Linux showed a failure in a test that was not running before. It looks like we may be able to avoid the manual test manifest for Linux moving forward at least https://oleb.net/2020/swift-test-discovery/. For whatever reason on Linux this was normalized to `Last-Event-Id`. With Xcode it's just passed through, so I just changed it here rather than adding normalization to the tests.

Reply:
See https://github.com/launchdarkly/swift-eventsource/pull/28